worldedit-core/src/main/resources/lang/strings.json

```diff
@@ -107,6 +107,7 @@
     "worldedit.schematic.save.failed-directory": "Could not create folder for schematics!",
     "worldedit.schematic.save.saving": "(Please wait... saving schematic.)",
     "worldedit.schematic.save.still-saving": "(Please wait... still saving schematic.)",
+    "worldedit.schematic.share.unsupported-format": "The schematic share destination \"{0}\" does not support \"{1}\" schematics.",
```

Comment:
I think this leads to awkward/redundant messages like `does not support "SPONGE_SCHEMATIC" schematics`. Maybe just `does not support the \"{1}\" format.`?